ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -943,6 +955,10 @@ Status MarkForClustering(Graph* graph, const std::set<string> skip_these_nodes,
       type_constraint_map["Slice"]["Index"] = NGraphIndexDTypes();
       type_constraint_map["Snapshot"]["T"] = NGraphDTypes();
       type_constraint_map["Softmax"]["T"] = NGraphNumericDTypes();
+      // For SoftmaxCrossEntropyWithLogits, see
+      // https://github.com/tensorflow/tensorflow/blob/c95ca05536144451ef78ca6e2c15f0f65ebaaf95/tensorflow/core/ops/nn_ops.cc#L1096
+      type_constraint_map["SoftmaxCrossEntropyWithLogits"]["T"] =
+          NGraphNumericDTypes();
```

Comment:
```suggestion
          NGraphRealDTypes();
```

Reply:
Agreed. Should we then also update the similar one for  `type_constraint_map["SparseSoftmaxCrossEntropyWithLogits"]["T"] =` ... ? @sayantan-nervana

Reply:
Yes.

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/ops.pbtxt#L44100
T-> NGraphRealDTypes
Tlabel-> NGraphIndexDTypes